src/base/bittorrent/peerinfo.cpp

```diff
@@ -367,6 +367,10 @@ void PeerInfo::determineFlags()
     if (useUTPSocket())
         updateFlags(u'P', C_UTP);
 
+    // h = Peer is using UDP hole punching
```

Comment:
Why do the following have discrepancies?
1. UDP hole punching
2. NAT hole punch

Reply:
IIRC, it should be called "NAT Hole Punching".

Reply:
> 1. UDP hole punching

That's what the [wiki](https://en.wikipedia.org/wiki/UDP_hole_punching) calls it, so that's why I made the comment that way. It is more important for the user to see that they are bypassing the NAT ISP, so the description is different.

Reply:
In general, it is called [NAT hole punching](https://en.wikipedia.org/wiki/NAT_hole_punching) (as I stated above). UDP hole punching, TCP hole punching etc. are just varieties of it.

Reply:
Changed.

Reply:
> Changed.

But how about second mentioned string? It is still "NAT hole punch" instead of "NAT hole punching". I've never seen it named as "NAT hole punch", but only as "NAT hole punching". Although "punch" and "punching" are essentially different forms of the same word, I prefer to stick to the commonly used one.